Enhance bulk edit login

hurradieweltgehtunter opened the pull request:
Follow-up for https://github.com/owncloud/web/pull/8799

## Description
Re-use the select box label from the right sidebar sidebar ("Login") instead of introducing a new one ("Login status") which saves an additional translation and stays consistent with editing a single user in the right sidebar.

Also, add an informational text line in the modal to explain what happens here.

## Screenshots (if appropriate):
### Before
<img width="516" alt="grafik" src="https://user-images.githubusercontent.com/16665512/232786572-6029f5f1-4875-4c8b-bb79-5fea616a7861.png">

### After
<img width="520" alt="grafik" src="https://user-images.githubusercontent.com/16665512/232786398-edb865c7-a309-47e3-8b9c-9fd33d507195.png">

JammingBen commented:
We recently tried to unify modals and removed texts above input fields/moved such texts to the modal title (see https://github.com/owncloud/web/issues/8579). IMO the text here doesn't justify to break with this approach.

"Login status" -> "Login" I agree, then we have it consistent with the right sidebar 👍

hurradieweltgehtunter commented:
> We recently tried to unify modals and removed texts above input fields/moved such texts to the modal title (see #8579). IMO the text here doesn't justify to break with this approach.
> 
> "Login status" -> "Login" I agree, then we have it consistent with the right sidebar 👍

Alright, didn't knew that. Removed it.

hurradieweltgehtunter commented:
hm build fails because of documentation `Unable to locate config file`. What to do here?

kulmann commented on packages/web-app-admin-settings/src/components/Users/LoginModal.vue:
Don't know if this is a rebase-gone-wrong, but I don't see a code change for the new message in the modal. Only the changed label string.

Judging by the screenshot, please use `$ngettext` and provide different strings for a single selected user or multiple selected users.

hurradieweltgehtunter replied:
Message got removed based on a comment by @JammingBen https://github.com/owncloud/web/pull/8853#issuecomment-1513150489

kulmann replied:
ah ok, thanks. makes sense!
